spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -131,7 +167,7 @@ module.exports = function (provider, setup) {
 
         // Wait a second before unpausing.  Data received in this second
         // will fail the expectation that paused is false in gotMessageAsync.
-        setTimeout(function() {
+        setTimeout(function () {
```

Comment:
Do we want to try to resolve this differently now, or is it okay to go ahead and merge this pull request? I think the other outstanding issues are resolved (though do correct me if I'm wrong).